src/main/java/com/dev/cinema/Main.java

```diff
@@ -52,11 +54,10 @@ public static void main(String[] args) throws AuthenticationException {
 
         shoppingCartService.registerNewShoppingCart(user);
         shoppingCartService.addSession(session, user);
-        System.out.println(shoppingCartService.getByUser(user));
+        logger.info(shoppingCartService.getByUser(user));
 
         orderService.completeOrder(shoppingCartService.getByUser(user).getTickets(), user);
-        System.out.println(shoppingCartService.getByUser(user));
-        System.out.println(orderService.getOrderHistory(user));
+        logger.info(shoppingCartService.getByUser(user));
+        logger.info(orderService.getOrderHistory(user));
```

Comment:
You can use some extra info in logs (that was mentor suggestion to a similar situation in my PR)

Reply:
I agree, let's make the logs more informative

Reply:
done